services/manifest-repo-export-service/pkg/cmd/server.go

```diff
@@ -215,10 +215,13 @@ func Run(ctx context.Context) error {
 				return fmt.Errorf("error in readEslEvent %v", err)
 			}
 			if ddMetrics != nil {
-				processDelay, err := calculateProcessDelay(ctx, esl, time.Now())
+				processDelay, err := calculateProcessDelay(ctx, esl, time.Now().UTC())
 				if err != nil {
 					log.Error("Error in calculateProcessDelay %v", err)
 				}
+				if processDelay < 0 {
+					log.Warn("process delay is negative: esl-time: %v, now: %v, delay: %v", esl.Created, time.Now().UTC(), processDelay)
```

Comment:
```suggestion
                                now :=time.Now().UTC()
				processDelay, err := calculateProcessDelay(ctx, esl, now)
				if err != nil {
					log.Error("Error in calculateProcessDelay %v", err)
				}
				if processDelay < 0 {
					log.Warn("process delay is negative: esl-time: %v, now: %v, delay: %v", esl.Created, now, processDelay)
```

You need to use one variable in both functions, otherwise the numbers don't match.